helm/README.md.gotmpl

```diff
@@ -0,0 +1,21 @@
+{{- /*
```

Comment:
Why do we need this file?

Reply:
This file defines the layout of the README.  Tried to improve its appearance, but it was unsuccessful. It will be removed.